build-scripts/test-results.cake

```diff
@@ -0,0 +1,217 @@
+// ***********************************************************************
+// Copyright (c) Charlie Poole and TestCentric GUI contributors.
+// Licensed under the MIT License. See LICENSE.txt in root directory.
+// ***********************************************************************
+
+// This file contains classes used to interpret the result XML that is
+// produced by test runs of the GUI.
+
```

Comment:
Actually what's this file for, sorry? It seems to just be returning a single number - although I'm presuming there's a reason for this, rather than just checking the console exit code?